scripts/scripts.js

```diff
@@ -71,6 +71,105 @@ export function createTag(tag, attributes, html) {
   return el;
 }
 
+/**
+ *Example
+ *
+  'html|button|
+  class:license-user-remove-user action important,type:button,data-new-productuserid:dataResponse0|
+  new_productuserid|
+  Remove User'
+
+ *Format
+
+  'html|htmlTagName|
+   attribute1:attribute1Value, attribute2: attribute2Value, attribute3: dataResponse0|
+   dataResponse0Key,dataResponse1Key|htmlInnerText
+
+  *In the above format dataResponse0 denotes take the key from next split
+  * Currently only used by create generic table method, export to use for others and update this comment.
+*/
+function processTag(mapping, row) {
```

Comment:
I suggest chaning mapping to a JSON object to make this easer to use and the following code easier to understand.

Reply:
Removed this method